[0m11:37:31.009622 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1049f1f60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1062fd600>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1062fcee0>]}


============================== 11:37:31.018157 | ddb78db0-0bc6-42f9-888d-b89ef98f1803 ==============================
[0m11:37:31.018157 [info ] [MainThread]: Running with dbt=1.8.6
[0m11:37:31.018415 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/joel/.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m11:37:31.042045 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/Users/joel/.dbt')}
[0m11:37:31.042299 [info ] [MainThread]: Creating dbt configuration folder at 
[0m11:37:42.345487 [debug] [MainThread]: Starter project path: /Users/joel/Documents/projects/dbt-tutorial/.venv/lib/python3.10/site-packages/dbt/include/starter_project
[0m11:37:42.360752 [info ] [MainThread]: 
Your new dbt project "temp" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m11:37:42.361390 [info ] [MainThread]: Setting up your profile.
[0m11:37:47.985258 [info ] [MainThread]: Profile temp written to /Users/joel/.dbt/profiles.yml using target's sample configuration. Once updated, you'll be able to start developing with dbt.
[0m11:37:47.985675 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 17.003702, "process_user_time": 0.553509, "process_kernel_time": 0.07876, "process_mem_max_rss": "121077760", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m11:37:47.985972 [debug] [MainThread]: Command `dbt init` succeeded at 11:37:47.985925 after 17.00 seconds
[0m11:37:47.986166 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1049f1f60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106367790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1063657b0>]}
[0m11:37:47.986357 [debug] [MainThread]: Flushing usage events
[0m12:42:07.919307 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103089f90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104995630>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104994f10>]}


============================== 12:42:07.921812 | e81f7487-65a2-4f4b-b711-bb91b07ef07d ==============================
[0m12:42:07.921812 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:42:07.922105 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/joel/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m12:42:13.780185 [debug] [MainThread]: Starter project path: /Users/joel/Documents/projects/dbt-tutorial/.venv/lib/python3.10/site-packages/dbt/include/starter_project
[0m12:42:13.794886 [info ] [MainThread]: 
Your new dbt project "temp_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m12:42:13.795544 [info ] [MainThread]: Setting up your profile.
[0m12:42:19.283115 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 11.391198, "process_user_time": 0.510624, "process_kernel_time": 0.082913, "process_mem_max_rss": "111362048", "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:42:19.283470 [debug] [MainThread]: Command `dbt init` succeeded at 12:42:19.283423 after 11.39 seconds
[0m12:42:19.283647 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103089f90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103dd54e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104994490>]}
[0m12:42:19.283820 [debug] [MainThread]: Flushing usage events
[0m14:15:46.949566 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x102a25f90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10433ca60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10433c340>]}


============================== 14:15:46.951826 | 2addab86-158e-403d-a45b-68a94fee21a2 ==============================
[0m14:15:46.951826 [info ] [MainThread]: Running with dbt=1.8.6
[0m14:15:46.952110 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/Users/joel/.dbt', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m14:15:46.952365 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/joel/Documents/projects/dbt-tutorial/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m14:15:46.952623 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.029973708, "process_user_time": 0.44409, "process_kernel_time": 0.051721, "process_mem_max_rss": "94601216", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m14:15:46.952897 [debug] [MainThread]: Command `dbt run` failed at 14:15:46.952855 after 0.03 seconds
[0m14:15:46.953066 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x102a25f90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10433c760>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10433cd90>]}
[0m14:15:46.953219 [debug] [MainThread]: Flushing usage events
[0m17:41:35.003905 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x102769f30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040757e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040750c0>]}


============================== 17:41:35.005932 | 8fba2d79-9fe5-4ec6-9323-07374a3c8778 ==============================
[0m17:41:35.005932 [info ] [MainThread]: Running with dbt=1.8.6
[0m17:41:35.006200 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/Users/joel/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt docs generate', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m17:41:35.006476 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/joel/Documents/projects/dbt-tutorial/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m17:41:35.006742 [debug] [MainThread]: Resource report: {"command_name": "generate", "command_wall_clock_time": 0.031010168, "process_user_time": 0.455081, "process_kernel_time": 0.046094, "process_mem_max_rss": "94912512", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m17:41:35.007085 [debug] [MainThread]: Command `dbt docs generate` failed at 17:41:35.007046 after 0.03 seconds
[0m17:41:35.007243 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x102769f30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040754e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104075b10>]}
[0m17:41:35.007381 [debug] [MainThread]: Flushing usage events
[0m09:41:05.361927 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104ec5fc0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1067e0af0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1067e03d0>]}


============================== 09:41:05.364233 | be56e7b2-5a6f-4c5c-b6a8-85de2509f6d6 ==============================
[0m09:41:05.364233 [info ] [MainThread]: Running with dbt=1.8.6
[0m09:41:05.364516 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/joel/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt run', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m09:41:05.364779 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/joel/Documents/projects/dbt-tutorial/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m09:41:05.365225 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.03078775, "process_user_time": 0.455821, "process_kernel_time": 0.042365, "process_mem_max_rss": "92979200", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m09:41:05.365484 [debug] [MainThread]: Command `dbt run` failed at 09:41:05.365448 after 0.03 seconds
[0m09:41:05.365636 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104ec5fc0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1067e07f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1067e37c0>]}
[0m09:41:05.365776 [debug] [MainThread]: Flushing usage events
[0m12:17:04.991918 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1045e5f30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105ef17e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105ef10c0>]}


============================== 12:17:04.994231 | ba05d723-fce1-4481-a6a7-8a24125c0998 ==============================
[0m12:17:04.994231 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:17:04.994512 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/joel/.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt docs generate', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m12:17:04.994771 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/joel/Documents/projects/dbt-tutorial/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:17:04.995027 [debug] [MainThread]: Resource report: {"command_name": "generate", "command_wall_clock_time": 0.031009791, "process_user_time": 0.455443, "process_kernel_time": 0.057506, "process_mem_max_rss": "94994432", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:17:04.995319 [debug] [MainThread]: Command `dbt docs generate` failed at 12:17:04.995283 after 0.03 seconds
[0m12:17:04.995470 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1045e5f30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105ef14e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105ef1b10>]}
[0m12:17:04.995615 [debug] [MainThread]: Flushing usage events
[0m12:30:31.281231 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1029c1f60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104e84a90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104e84370>]}


============================== 12:30:31.283225 | 81b5cf3a-7c77-42dc-87b4-d72a52473405 ==============================
[0m12:30:31.283225 [info ] [MainThread]: Running with dbt=1.8.6
[0m12:30:31.283517 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/joel/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt test', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m12:30:31.283783 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/joel/Documents/projects/dbt-tutorial/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m12:30:31.284034 [debug] [MainThread]: Resource report: {"command_name": "test", "command_wall_clock_time": 0.027163958, "process_user_time": 0.46791, "process_kernel_time": 0.046687, "process_mem_max_rss": "95010816", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m12:30:31.284286 [debug] [MainThread]: Command `dbt test` failed at 12:30:31.284253 after 0.03 seconds
[0m12:30:31.284431 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1029c1f60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104e84790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104e87760>]}
[0m12:30:31.284568 [debug] [MainThread]: Flushing usage events
